modules/database/src/adapters/sequelize-adapter/index.ts

```diff
@@ -255,15 +256,24 @@ export class SequelizeAdapter extends DatabaseAdapter<SequelizeSchema> {
   }
 
   async ensureConnected() {
-    return this.sequelize
-      .authenticate()
-      .then(() => {
+    let error;
+    for (let i = 0; i < 10; i++) {
+      i === 0
+        ? ConduitGrpcSdk.Logger.log('Attempting to connect to database...')
```

Comment:
We probably shouldn't be that verbose about this. Just log the attempt once, then log failure if all attempts fail.